components/combobox/__examples__/snapshot/base-label-required.jsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-console, react/prop-types */
```

Comment:
I'm not 100% sure, but I think if you don't need the menu open you can just import the "non-snapshot" story into here, since it will look the same.

Reply:
Makes sense - I reused the "non-snapshot" version here.